### DISCUSSION TITLE
386 Max

### DISCUSSION THREAD

**Discussion by stuaxo** [dosemu2/dosemu2#1883] [General]
I only just  found out that The 386 Max memory manager went GPL 6 months ago [1], after the original author realised the company he sold it to had gone bust and the rights reverted to him.

I'm not sure if this is useful to DOSEMU2 or any of the CI and tests or not though.

[1] Mentioned in this interview with Jim Hall https://www.theregister.com/2023/01/18/retro_tech_week_freedos/?page=2

**Comment by stsp** [comment]
https://github.com/sudleyplace/386MAX
Not sure if it is any helpful here.
